### ISSUE TITLE
change grinffin-lim algorithm to wavenet-based vocoder

### ISSUE THREAD

**Issue by 493238731** [keithito/tacotron#64]
Does anyone plan to use wavenet-based vocoder, insteading of griffin-lim algoritm, which greatly increase the audio quality in the single-speaker experiment of [Deep Voice 2](https://arxiv.org/abs/1705.08947). 
How can we  change [Wavenet](https://github.com/ibab/tensorflow-wavenet) to that which Deep Voice 2 shows

**Comment by feiyuhong** [comment]
I've been trying to use wavenet insteading of  griffin-lim recently .

**Comment by 493238731** [comment]
@feiyuhong Finish it? Can you show me some experienment result and loss curve?

**Comment by rafaelvalle** [comment]
@feiyuhong can you share link to some code? I'm interested in porting it to pytorch!

**Comment by lsq357** [comment]
In my result, I can't get the clear voice after a week training when batch_size=32
Also, it is too slow that only train 1.6k+ step each day in my single GTX1080ＴＩ ＧＰＵ, I need more GPUs.

**Comment by toannhu** [comment]
@keithito Do you plan to use Wavenet Vocoder with Tacotron? There is a repo to do this task https://github.com/r9y9/wavenet_vocoder. Hope to see the good quality sound just like the white paper did.

**Comment by keithito** [comment]
@toannhu Yes, that repo looks great! I'm training right now on LJ Speech. There's some more discussion over at https://github.com/keithito/tacotron/issues/90

**Comment by mrgloom** [comment]
I wonder why Griffin-Lim samples from here https://nv-adlr.github.io/WaveGlow have as good quality as other methods? Is it large number of iterations or it's just good output of tacotron?
